package.json

```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/YorkAARGH/Misaki#readme",
   "dependencies": {
+    "base-64": "^0.1.0",
```

Comment:
As written before, this dependency could be removed and avoided if you don't specify `encode=base64` in the API.